cds-snc/cppd-medical-report-sails
# Add validation library and helper

This PR adds a validation library and helper for validating requests in a controller.  

This gets the basics working - the ability to pass a request object through a validator with a set of rules. The validator will then flash any error messages and form data to the session and then redirect back in the case of errors, otherwise it will return control to the controller method which can then save data, optionally set a success message, then redirect to the next step or confirmation.

Note that the original macros/templates from the Node Starter were designed to handle a very specific error object format. Changes to accommodate will come in a separate PR.

## Adds:
- Validate.js validation library
- Set some basic validation on Personal form for testing
- Render validation errors at top of form

## Missing:
- Validation message and styling on individual form elements. This should be done in a separate PR.

## To test:
- Navigate to the /en/personal page
- Submit the form without entering anything
- Notice the list of validation messages at the top of the form
- You will not see styling/messages on individual form elements
